utils/email_service.py

```diff
@@ -0,0 +1,7 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from decouple import config
+
+@shared_task
+def send_email(recipient,subject,body):
+    send_mail(subject, body, config('EMAIL_HOST_USER') , [recipient,])
```

Comment:
```suggestion
    send_mail(subject, body, config('EMAIL_HOST_USER') , recipient)
```

let the users enter a list of recipients

Reply:
When i try to send html templated mail it doesn't work. It send the mail as text. 

![image](https://user-images.githubusercontent.com/33046846/81907289-26400280-95e5-11ea-808e-dffedc764a75.png)

Please try something like this. We will be send html email only. :innocent: 

![image](https://user-images.githubusercontent.com/33046846/81907383-48d21b80-95e5-11ea-969a-186c4456a4f8.png)

Reply:
Made the changes, please check again.